e2e/e2e_docker_test.go

```diff
@@ -17,20 +17,25 @@
 package e2e_test
```

Comment:
`e2e_docker_test` doesn't tell us what it's testing. We're not testing docker here.

Do you mean e2e/fn_test.go?

Reply:
Will do it in the same PR for previous comment.